meta-genivi-dev/meta-genivi-dev/recipes-web/zebkit/zebkit_1.0.0+20170516.bb

```diff
@@ -0,0 +1,293 @@
+#   node_modules/browser-env/node_modules/window/node_modules/jsdom/node_modules/html-encoding-sniffer/node_modules/whatwg-encoding/LICENSE.txt
+#   node_modules/browser-env/node_modules/window/node_modules/jsdom/node_modules/whatwg-encoding/LICENSE.txt
+#   node_modules/browser-env/node_modules/window/node_modules/jsdom/node_modules/xml-name-validator/LICENSE.txt
+#   node_modules/browser-env/node_modules/window/node_modules/jsdom/node_modules/content-type-parser/LICENSE.txt
+#   node_modules/browser-env/node_modules/window/node_modules/jsdom/node_modules/sax/LICENSE
+#   node_modules/browser-env/node_modules/window/node_modules/jsdom/node_modules/escodegen/LICENSE.source-map
+#   node_modules/browser-env/node_modules/window/node_modules/jsdom/node_modules/escodegen/LICENSE.BSD
+#   node_modules/browser-env/node_modules/window/node_modules/jsdom/node_modules/escodegen/node_modules/esprima/LICENSE.BSD
+#   node_modules/browser-env/node_modules/window/node_modules/jsdom/node_modules/escodegen/node_modules/source-map/LICENSE
+#   node_modules/browser-env/node_modules/window/node_modules/jsdom/node_modules/escodegen/node_modules/source-map/node_modules/amdefine/LICENSE
+#   node_modules/browser-env/node_modules/window/node_modules/jsdom/node_modules/escodegen/node_modules/esutils/LICENSE.BSD
+#   node_modules/browser-env/node_modules/window/node_modules/jsdom/node_modules/escodegen/node_modules/estraverse/LICENSE.BSD
+#   node_modules/browser-env/node_modules/window/node_modules/jsdom/node_modules/webidl-conversions/LICENSE.md
+#   node_modules/browser-env/node_modules/window/node_modules/jsdom/node_modules/request/LICENSE
+#   node_modules/browser-env/node_modules/window/node_modules/jsdom/node_modules/request/node_modules/hawk/LICENSE
+#   node_modules/browser-env/node_modules/window/node_modules/jsdom/node_modules/request/node_modules/hawk/node_modules/boom/LICENSE
+#   node_modules/browser-env/node_modules/window/node_modules/jsdom/node_modules/request/node_modules/hawk/node_modules/boom/node_modules/hoek/LICENSE
+#   node_modules/browser-env/node_modules/window/node_modules/jsdom/node_modules/request/node_modules/hawk/node_modules/hoek/LICENSE
+#   node_modules/browser-env/node_modules/window/node_modules/jsdom/node_modules/request/node_modules/hawk/node_modules/cryptiles/LICENSE
+#   node_modules/browser-env/node_modules/window/node_modules/jsdom/node_modules/request/node_modules/hawk/node_modules/cryptiles/node_modules/boom/LICENSE
+#   node_modules/browser-env/node_modules/window/node_modules/jsdom/node_modules/request/node_modules/hawk/node_modules/cryptiles/node_modules/boom/node_modules/hoek/LICENSE
+#   node_modules/browser-env/node_modules/window/node_modules/jsdom/node_modules/request/node_modules/hawk/node_modules/sntp/LICENSE
+#   node_modules/browser-env/node_modules/window/node_modules/jsdom/node_modules/request/node_modules/hawk/node_modules/sntp/node_modules/hoek/LICENSE
```

Comment:
@frznlogic  can you see it again. I have fixed the licensing stuff in the recipe.